pageserver/src/tenant_threads.rs

```diff
@@ -69,6 +81,12 @@ pub fn wait_for_tenant_threads_to_stop(tenantid: ZTenantId) {
 /// Checkpointer thread's main loop
 ///
 fn checkpoint_loop(tenantid: ZTenantId, conf: &'static PageServerConf) -> Result<()> {
+    let gauge = TENANT_THREADS_COUNT.with_label_values(&["checkpointer"]);
```

Comment:
NIT: Feels like we could move both of those gauge fiddlings one level above, into `spawn(move || {` block, since in the loop functions not much of a thread as a context left.